src/Pretzel.Logic/Templating/Context/PathExtensions.cs

```diff
@@ -5,7 +5,7 @@ namespace Pretzel.Logic.Templating.Context
 {
     public static class PathExtensions
     {
-        private static readonly string[] MarkdownFiles = new[] { ".md", ".mdown", ".markdown" };
+        private static readonly string[] MarkdownFiles = new[] { ".md", ".mkd", ".mkdn", ".mdown", ".markdown" };
```

Comment:
Thanks for the review!
I think that these extensions are the most used, so it is may be not necessary to add the others?
Or "mdwn" only.